server/data/documentsApiClient.ts

```diff
@@ -0,0 +1,76 @@
+import superagent from 'superagent'
+import RestClient from './restClient'
+import config from '../config'
+import DocumentsApiClient from './interfaces/documentsApi/DocumentsApiClient'
+import DocumentMeta from './interfaces/documentsApi/DocumentMeta'
+import logger from '../../logger'
+import DocumentSearchResponse from './interfaces/documentsApi/DocumentSearchResponse'
+
+export default class DocumentsApiRestClient implements DocumentsApiClient {
+  private readonly restClient: RestClient
+
+  constructor(private readonly token: string) {
+    this.restClient = new RestClient('Documents Service API', config.apis.documentsServiceApi, token)
+  }
+
+  async putProfilePhoto(
+    document: Buffer,
+    metadata: Record<string, string>,
+    uuid: string,
+    activeCaseLoadId: string,
+    username: string,
+  ): Promise<DocumentMeta> {
+    try {
+      const result = await superagent
+        .post(`${config.apis.documentsServiceApi.url}/documents/PRISONER_PROFILE_PICTURE/${uuid}`)
+        .retry(2, (err, res) => {
+          if (err) logger.info(`Retry handler found API error with ${err.code} ${err.message}`)
+          return undefined // retry handler only for logging retries, not to influence retry logic
+        })
+        .auth(this.token, { type: 'bearer' })
+        .set('service-name', 'hmpps-prisoner-profile')
+        .set('Active-Case-Load-Id', activeCaseLoadId)
```

Comment:
This is an interesting way of capturing this... I might steal this for the Prison Person API to save making another API call to capture it...